src/main/java/eu/dissco/exportjob/domain/TargetType.java

```diff
@@ -20,4 +21,14 @@ public enum TargetType {
     this.name = name;
   }
 
+  public static TargetType fromString(String s){
+    if ("https://doi.org/21.T11148/894b1e6cad57e921764e".equals(s)){
```

Comment:
Worry a bit about the hardcoded type, we could get new type handles (or DOIs?) for prod right? Not blocking but might be something to check if we get new type handles